puphpet/config.yaml.template

```diff
@@ -60,10 +71,20 @@ vagrantfile-local:
         host: detect
 server:
     install: '1'
+    apt-ppa-repos:
+        - ppa:git-core/ppa   # Provides newer git-core. -bp
+        - ppa:chris-lea/node.js  # Provides newer node and npm. -bp
     packages:
-        - vim
-        - memcached
         - curl
+        - emacs
```

Comment:
I didn't say _using_ vagrant to produce production boxes, I said we could _match_ a fixed set of tools in a static production image (like an AMI for instance) and it's beneficial to have solid parity across the two. 

I'm not sure I can explain myself any more clearly that the punishment is meant to be for _not doing your job properly._ (By not properly customizing a VM for the given project, whether you're removing default packages or adding them, it's the same point.)

And I still have pretty strong reservations about not being able to "depend" on things. The set of packages I have included mean that we can move plugins and code between projects and have a reasonable guarantee that they'll work without any boobytraps. We should just always be able to count on being able to set `Expires` headers via Apache, for example, but that's not true unless `mod_expires` is consistently enabled everywhere. Not having this guarantee leads to more (duplicated and unnecessary) troubleshooting sessions trying to figure out why something doesn't work.) Things should Just Work :tm:, and the only way to guarantee that is to include the packages are are **likely** to be relevant, whether they immediately are or not. A slower-provisioning VM is a super small price to be paid for that benefit, based on my (extensive) experience.

That experience also tells me that changing the packages in the config.yaml isn't a sure thing. Sometimes there are conflicts or other sub-dependencies, and troubleshooting those will waste your entire day. The list I included is battle-tested and stable. Nobody will have to sacrifice their time again working out the right incantation to make the VM work properly. Why throw that away?

That said, here's what reluctantly changed my mind:

There are two approaches being suggested here: 

1. Leave everything enabled; force devs to delete unnecessary packages. Penalty is a slow provisioning VM, which is a time waster whenever you rebuild a VM.
1. Leave nothing enabled; force devs to add packages as necessary. Penalties are that there is no good reference of which packages to "prefer", you might have to hunt for the correct package name(s) to add, and that you might not even realize you're missing something you need right away, all of which is a time waster every time you deal with code that depends on a potentially-missing package.

Given the fact that a "new" project kind of by definition can't yet "depend" any of the packages in its fresh state (there is no code that demands the presence of memcache, for example [yet]), and the fact that in both cases you have to **edit the config.yaml file anyway** it might as well be to turn on commented out packages instead of turning off active ones. 

But, two caveats:

1. My list stays in the config, just commented out. There's too much valuable experience encoded in it to throw it away.
1. I don't want to argue about is what still stays enabled and what gets disabled. I think we could spend forever in another tabs vs spaces debate about which packages should be considered "mandatory" defaults. The only rule is: **Does the skeleton app need it to run "out of the box?"** For example, see #81. That means no vim or emacs either, since we can't predict which one **might** be preferred on a given project. 

I've added a new commit that disables everything not required for the current skeleton to run properly.